examples/client-logger/app-insights-logger/README.md

```diff
@@ -0,0 +1,15 @@
+# @fluid-example/app-insights-logger
```

Comment:
I recommend checking out tools/api-markdown-documenter/README.md. It takes advantage of a number of our template macros for baking in various bits of useful information like our trademark info, contribution guidelines, etc. Some of our other sample app packages also use a macro for including app launching instructions that might be useful here.

Reply:
Note: in order to generate the contents, you'll have to run a build from the docs directory (for now).